### ISSUE TITLE
Add Foldable Device Support

### ISSUE THREAD

**Issue by nojronatron** [nojronatron/Bigfoot-Bib-Report-WL-Form#112]
**Describe the bug**

Split and foldable screens work, however they display bars that can obscure content in some situations. This is a display-only bug that reduces UX and possibly interferes with data entry or readability.

**To Reproduce**

Steps to reproduce the behavior on a real foldable device:

1. Open the form on a foldable device
2. Fold the device (or expand to dual-screen mode, depending on the device)
3. Note the dark vertical bar that appears down the center of the screen

Steps to reproduce the behavior on a simulated device (Chrome demonstrated, others have similar steps):

1. Open form with a web browser
2. Open Developer Tools
3. Click `Toggle device toolbar`
4. Select one of `[ Surface Duo, Galaxy Z Fold 5, Asus Zenbook Fold, ...]`
5. Click `Toggle dual-screen mode` button (or `Continuous/Folded` dropdown, depending on the device selected in step 4)
6. Optional: Click `Screen Orientation Options` to rotate the device screen

**Expected behavior**

Opening the page on a folded (or dual-screen mode) device displays a full, usable form page with no obscured sections or content.

**Screenshots**

<img width="1105" height="804" alt="Surface Duo Dual Screen Mode (vertical)" src="https://github.com/user-attachments/assets/50b8bb2b-98db-4e7b-87a5-f239326d769a" />

<img width="798" height="638" alt="Surface Duo Dual Screen Mode (horizontal)" src="https://github.com/user-attachments/assets/4c56ce28-6743-4383-af5c-4d1041be8883" />

<img width="892" height="722" alt="Asus Zenbook Fold (horizontal)" src="https://github.com/user-attachments/assets/a3b3f4aa-9eea-4be3-a93d-40c5caae6999" />

**Mobile Devices**
 - Devices: Galaxy Z Fold, Zenbook Fold, Surface Duo

**Additional context**

Do one or more of the following to work around the obscuring dark bar:

- Flatten the foldable device to get rid of the dark/black bar
- Toggle single-screen mode
- Adjust the orientation from vertical to horizontal (or vice-versa) to move content out from under the obscured fold or screen-split bar

**Comment by nojronatron** [comment]
Check out [Using Viewport Segments API](https://developer.mozilla.org/en-US/docs/Web/API/Viewport_segments_API/Using)
